app/controllers/hiring_staff/vacancies/job_specification_controller.rb

```diff
@@ -50,17 +50,29 @@ def update
   private
 
   def job_specification_form_params
-    params.require(:job_specification_form).permit(:job_title, :job_description, :leadership_id,
-                                                   :subject_id,
+    persist_nqt_job_role_to_nqt_attribute
+    params.require(:job_specification_form).permit(:job_title, :job_description, :subject_id,
                                                    :starts_on_dd, :starts_on_mm,
                                                    :starts_on_yyyy, :ends_on_dd, :ends_on_mm, :ends_on_yyyy,
                                                    :flexible_working, :newly_qualified_teacher,
                                                    :first_supporting_subject_id, :second_supporting_subject_id,
-                                                   working_patterns: [])
+                                                   working_patterns: [], job_role: [])
+  end
+
+  # Only necessary until changes to search are implemented
+  # TODO remove after migration to remove newly qualified teacher column
+  def persist_nqt_job_role_to_nqt_attribute
```

Comment:
Could you abstract this please?  It's identical to the method with the same name in the copy controller except that it uses a different parameter for the job_role and adds data to a different set of parameters, both of which could be passed in.